### ISSUE TITLE
Project version for 2.46.0 tag is not updated in MAPL/CMakeLists.txt

### ISSUE THREAD

**Issue by natalie-perlin** [GEOS-ESM/MAPL#2818]
./MAPL/CMakeLists.txt file for the recent 2.46.0 tag is not updated, and still says 2.45.0:
https://github.com/GEOS-ESM/MAPL/blob/54a2b20db47929181c3debcd7e13a99e2bb46d4b/CMakeLists.txt#L11

```
project (
  MAPL
  VERSION 2.45.0
  LANGUAGES Fortran CXX C)  # Note - CXX is required for ESMF
  ```

**Comment by mathomp4** [comment]
Craaaaaap. Bad Matt. I'll consult with @tclune on if we should push the tag or issue 2.46.1. I think pushing the tag is probably okay since no one is really using the commit hash and I haven't yet updated the spack package.

Thanks for noticing this.

**Comment by mathomp4** [comment]
@natalie-perlin Do you have any objection to me pushing the v2.46.0 tag to a newer commit? If not, I might just do that.

**Comment by mathomp4** [comment]
Welp, I've re-released MAPL v2.46.0. For the sake of artifacts, etc. I deleted the old v2.46.0 tag and release, fixed the version, and then re-released MAPL 2.46.0.

Thanks for pointing this out!

**Comment by natalie-perlin** [comment]
Thank you @mathomp4 !
